### PR TITLE
Clan status setting

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: web/templates/settings/clansettings.html [hunk 1]
```diff
@@ -52,13 +53,23 @@
                     placeholder="Icon"
                     value="{{ $icon }}" />
                 </div>
+                <div class="field">
+                  <label class="left aligned">
+                    Clan Status
+                  </label>
+                  <select tabindex="4" name="status">
```
Comment (cmyui) [inline web/templates/settings/clansettings.html]:
```suggestion
                  <select tabindex="4" name="status" value="{{ $status }}">
```

File: web/templates/settings/clansettings.html [hunk 2]
```diff
@@ -21,10 +21,11 @@
               {{ $desc := or $d.description.String "" }}
               {{ $icon := or $d.icon.String "" }}
               {{ $bg := or $d.background.String "" }}
+              {{ $status := or $d.status.String "" }}
```
Comment (cmyui) [inline web/templates/settings/clansettings.html]:
```suggestion
              {{ $status := or $d.status.Int 0 }}
```

File: web/templates/settings/clansettings.html (hunk 1 above)
Comment (cmyui) [reply web/templates/settings/clansettings.html]: i don't think this is correct html but we basically need the existing value to be set as the default

File: web/templates/settings/clansettings.html (hunk 2 above)
Comment (cmyui) [reply web/templates/settings/clansettings.html]: untested but it's an int on api side https://github.com/osuAkatsuki/akatsuki-api/blob/master/app/v1/clan.go#L383

File: web/templates/settings/clansettings.html (hunk 1 above)
Comment (ch03e) [reply web/templates/settings/clansettings.html]:
```suggestion
                  <select class="ui dropdown" tabindex="4" name="status" data-cast="int">
```

**Review by infernalfire72 — APPROVED** [review]

**Comment by cmyui** [comment]
reality

**Comment by ch03e** [comment]
my pr got stolen

**Pull request merged**